app/models/setting/content.rb

```diff
@@ -76,7 +76,9 @@ def self.load_defaults
         self.set('default_location_puppet_content',
                  N_('Default Location where new Puppet content will be put upon Content View publish'),
                  nil, N_('Default Location Puppet content'), nil,
-                 :collection => proc { Hash[Location.unscoped.all.map { |loc| [loc[:title], loc[:title]] }] })
+                 :collection => proc { Hash[Location.unscoped.all.map { |loc| [loc[:title], loc[:title]] }] }),
+        self.set('expire_soon_days', N_('The number of days remaining in a subscription before we warn you about upgrading.'),
```

Comment:
I'm not fond of the description including "we". How about something more like this:

"The number of days remaining in a subscription before you will be notified about upgrading."